### ISSUE TITLE
empty LocalFolder after restart. local directory on other drive

### ISSUE THREAD

**Issue by der3b** [cleidigh/Localfolder-TB#79]
moved a lot of messages to new drive today, to new LocalFolder directories. type of storage is maildir.
after restart all moved mails/folders are gone.
in account settings for a new 'Archive' LocalFolder the local directory is: L:\mail\archive
mails and folders are there, eg. Archieven.msf and Archieven.sbd folder

it looks like Thunderbird looks at C:\Users\MyName\AppData\Roaming\Thunderbird\Profiles\uffqf07c.default\Mail\Archive
same Archieven.msf and Archieven.sbd folder there, but no mails inside

any idea how to fix this situation without losing the mails/folders i moved?

**Comment by der3b** [comment]
details: 
OS = windows 10 pro, build 10.0.1945 
TB = 115.9.0 (64-bit)
LocalFolders = 4.0.0

**Comment by cleidigh** [comment]
@der3b 
I hope you didn't lose any mail with this...

I'm not completely clear on the sequence. Can you give the exact steps and the options when you created the local folder.
There is an issue when an LF is created in a non empty Directory. The chosen folders such as Archives get created and overwrite an existing folder. Because I am not clear on the sequence, I am not sure this applies.
@cleidigh

**Comment by der3b** [comment]
thanks for your reply. i made a new profile to check if i could redo it, not sure about sequence myself.
think i got it, i messed up two thinks.
i get an error when creating a new subdirectory ("Folder1")  in the windows explorer creating a LocalFolders account
"message: A subdirectory of the directory specified in the Local Directory setting is already used by the "Folder1" account. Please pick a different directory."
second is were i lost the view of the messages. i made a folder 'new' in the original 'Local Folders' and put one message in it. 
after creating "Folder1" i moved the folder 'new' to the "Folder1" account. the error i get is:
Error - new on folder1
"There is not enough disk space to download new messages. Try deleting old mail, emptying the Trash folder, and compacting your mail folders, and then try again."
i then created a new folder "0.new" in the "Folder1" account, no errors. then moved the "new" folder from  the original 'Local Folders' to the "0.new" folder, no errors. now the message is in "Folder1/0.new/new/message"
after this: exit TB, restart TB: no more folder '0.new' visible. on the hdd all is there
got the message back in sight by adding the '0.new' folder again.

bottom line:
 is restarting necessary after creating a new Local Folders account fixing this problem: yes

**Comment by cleidigh** [comment]
@der3b 
I will be honest , I didn't follow everything. I think I understand that you created some directions using file explorer. This could be part of the problem. Once a new Local Folder is made Thunderbird needs to manage it. Sometimes you can create a new folder in a LF folder, restart Thunderbird and Thunderbird will index and include it, this is not always the case.
Normally if you create an LF and subsequently create folders in it with Thunderbird and copy messages in with Thunderbird, everything should work.

I won't discount the possibility that new Thunderbird revisions might be doing something odd.
@cleidigh
